extract/text_line.py

```diff
@@ -0,0 +1,42 @@
+from .text import Text, get_text_from_xml_element, text_attrs_styles_are_equal
+import copy
+
+
+class TextLine():
+    """A class containing information from a <textline> node"""
+    attr = {}
+
+    texts = []
+
+    def __init__(self):
+        self.texts = []
+        self.attr = {}
```

Comment:
had some issues learning python class and object scopes, what i'm really after is attr and texts to both only be per-object scoped. Seems like I have to put it in the __init__ method for that to happen, but now I'm not sure if I also need to declare those two in the class as well?